tests/stavenote_tests.ts

```diff
@@ -9,6 +9,7 @@
 import { VexFlow } from '../src/vexflow';
 import { TestOptions, VexFlowTests } from './vexflow_test_helpers';
 
+import { Ornament } from '../src';
```

Comment:
This works but should be written as:

```
import { Ornament } from '../src/ornament';
```

I know the original was already wrong, but we might as well fix it.

Reply:
done